Makefile

```
@@ -109,6 +110,24 @@ cleanup:
 	rm -f .dirpath buf.yaml buf.lock
 	@if [ -f buf.work.yaml.bak ]; then mv buf.work.yaml.bak buf.work.yaml; fi
 
+generate-local: tools-check
```

checkmake (0.3.2)

[warning] Target body for "generate-local" exceeds allowed length of 5 lines (16).

(maxbodylength)